raft/test/raft_test.cpp

```diff
@@ -21,11 +21,11 @@ using namespace ::testing;
 
 namespace
 {
-    const bzn::uuid_t TEST_NODE_UUID{"f0645cc2-476b-485d-b589-217be3ca87d5"};
+    const bzn::uuid_t k_TEST_NODE_UUID{"f0645cc2-476b-485d-b589-217be3ca87d5"};
```

Comment:
Not sure we need the k_ as all caps usually indicates const and our compiler will tell us so if we try to modify it.

Reply:
I will remove the k_ 's